databricks/koalas/generic.py

```diff
@@ -311,6 +312,59 @@ def abs(self):
         # TODO: The first example above should not have "Name: abs(0)".
         return _spark_col_apply(self, F.abs)
 
+    def clip(self, lower: Union[float, int] = None, upper: Union[float, int] = None) \
+            -> Union['ks.Series', 'ks.DataFrame']:
+        """
+        Trim values at input threshold(s).
+
+        Assigns values outside boundary to boundary values. Thresholds can be singular values or
+        array like, and in the latter case the clipping is performed element-wise in the specified
+        axis.
+
+        Parameters
+        ----------
+        lower : float or int, default None
+            Minimum threshold value. All values below this threshold will be set to it.
+        upper : float or int, default None
+            Maximum threshold value. All values above this threshold will be set to it.
+
+        Returns
```

Comment:
add a small note section explaining the difference w.r.t. pandas.

Reply:
Added with 8f4876a.